src/psyclone/tests/lfric_ref_elem_test.py

```diff
@@ -173,8 +176,8 @@ def test_refelem_arglist_err():
     with pytest.raises(InternalError) as err:
         kernel.arguments.raw_arg_list()
     assert ("Unsupported reference-element property ('Not a property') found "
-            "when generating arguments for kernel 'testkern_ref_elem_code'. "
-            "Supported properties are: ['Property." in str(err.value))
+            "when generating kernel arguments. Supported properties are: "
```

Comment:
Is it possible to get the kernel name put back in this error message? (I appreciate it might not be due to restructuring but please have a look.)

Reply:
Yes, done.

Reply:
Thanks.